### PR TITLE
Fixes Issue #1

### PR DESCRIPTION
**Pull request by anonym0us3**
Fixes Issue #1 - made sure sidebar title fonts are all monospace

### CODE REVIEW

**Comment by tgaff** [comment]
@anonym0us3 I'm seeing merge conflicts here.

File: config/database.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ login: &login
   adapter: postgresql
   host: localhost
   port: 5432
+  username: postgres
+  password:
```
Comment (tgaff) [inline config/database.yml]: If you need this on your local system that's fine.  But please don't check it in... other open source users won't be expecting you to change the DB config.

**Comment by anonym0us3** [comment]
Fixes Issue #1. Resolved merge conflict, removed extraneous files, added 2nd fix to Issue #1: bullet-points

**Comment by tgaff** [comment]
LGTM :+1:

**Pull request closed without merging**